<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskNewPage</class>
 <widget class="QWidget" name="TaskNewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Page</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Main">
   <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelStandard">
        <property name="text">
         <string>Standard:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="standardComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSize">
        <property name="text">
         <string>Page Size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="sizeComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelOrientation">
        <property name="text">
         <string>Orientation:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayoutOrientation">
        <item>
         <widget class="QRadioButton" name="landscapeRadioButton">
          <property name="text">
           <string>Landscape</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="portraitRadioButton">
          <property name="text">
           <string>Portrait</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelLanguage">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="languageComboBox">
        <property name="toolTip">
         <string>Select language for translatable templates. Only ISO5457 templates are currently translatable.</string>
        </property>
       </widget>
      </item>
     </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPreview">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutPreview">
      <item>
       <widget class="QSvgWidget" name="svgPreviewWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addTemplateButton">
     <property name="text">
      <string>Add template</string>
     </property>
     <property name="toolTip">
      <string>Add custom svg templates. Clicking the button opens a file browser of the templates folder. There you can add a folder. You need to respect a folder hierarchy standard/size/portrait and landscape.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height> <!-- Adjusted spacer -->
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
    <class>QSvgWidget</class>
    <extends>QWidget</extends>
    <header>QtSvgWidgets/QSvgWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>